index.html

```diff
@@ -252,5 +252,66 @@ <h2 class="job">I’am a Software Developer</h2>
         </ul>
       </div>
     </div>
+    <!-- form section -->
+    <form action="https://formspree.io/f/mzbokejq" method="post">
+      <img class="image-form" src="image/Primary-Text.svg" alt="" />
+      <ul class="form-list">
+        <li class="form-name">
+          <label for="name" class="username"></label>
+          <input
+            type="text"
+            id="name"
+            name="user_name"
+            maxlength="30"
+            placeholder="Full Name"
+            required
+          />
+          <img class="form-name-img" src="image/Container.svg" alt="" />
+        </li>
+        <li class="form-email">
+          <label class="user-email" for="email"></label>
+          <input
+            type="email"
+            id="email"
+            name="user_email"
```

Comment:
- [ ] Kindly remove outlines of focus on each input as they are reducing the beauty of the design
<img width="341" alt="Screen Shot 2022-04-27 at 12 46 46" src="https://user-images.githubusercontent.com/28307960/165502037-9ef94cee-25a1-43b1-87d5-6716208bada3.png">